### DISCUSSION TITLE
mapsui.com

### DISCUSSION THREAD

**Discussion by EmmittS22** [Mapsui/Mapsui#2355] [Q&A]
Is there a problem with site mapsui.com as getting 404

![image](https://github.com/Mapsui/Mapsui/assets/154058126/21385d31-8e36-4ff7-b74c-73867e368152)

**Comment by pauldendulk** [comment]
We are migrating from our main branch from 'master' to 'main' today, I assume this is fixed after this PR is merged:
https://github.com/Mapsui/Mapsui/pull/2354
Funny how quickly you noticed.

**Reply by pauldendulk** [reply]
It is up again.

**Reply by EmmittS22** [reply]
Cheers Paul

**Comment by EmmittS22** [comment]
Cheers Paul



Sent from my Galaxy



-------- Original message --------
From: Paul den Dulk ***@***.***>
Date: 17/12/2023 11:49 (GMT+00:00)
To: Mapsui/Mapsui ***@***.***>
Cc: EmmittS22 ***@***.***>, Author ***@***.***>
Subject: Re: [Mapsui/Mapsui] mapsui.com (Discussion #2355)


It is up again.
